packages/story-editor/src/app/taxonomy/taxonomyProvider.js

```diff
@@ -67,7 +68,19 @@ function TaxonomyProvider(props) {
     (async function () {
       try {
         const result = await getTaxonomies();
-        setTaxonomies(Object.values(result));
+
+        const formattedTaxonomies = Object.values(result).map((taxonomy) => {
+          const entries = Object.entries(taxonomy);
+
+          const formattedEntries = entries.map((entry) => [
+            snakeCaseToCamelCase(entry[0]),
+            entry[1],
+          ]);
+
+          return Object.fromEntries(formattedEntries);
```

Comment:
I have a little util in `app/taxonomy/utils` that's called `mapObjectKeys()`. should be able to just:
```
const entries = mapObjectKeys(taxonomy, (key) => snakeCaseToCamelCase(key))
```

Reply:
Chatted with Max about this - this util is in a different package, so for now we're holding off on using it. In the future if this happens more we could move these utils to a shared package like the design system.